lib/server/controllers/dplDetectors.controller.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+const { updateExpressResponseFromNativeError } = require('../express/updateExpressResponseFromNativeError.js');
+const { getAllDplDetectors } = require('../services/dpl/getAllDplDetectors.js');
+
+/**
+ * Endpoint handler to list all dpl detectors
+ *
+ * @param {Object} _request the express request
```

Comment:
As we defined you can remove the params and simply put a description

Reply:
applied